home.html

```diff
@@ -100,34 +102,38 @@ <h4>
                     Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro assumenda sit non, nam in sint beatae. Veniam, mollitia ratione, explicabo ipsum ab minima nesciunt magnam, quis pariatur asperiores necessitatibus cumque.
                     <button class="btn">Read more <span><i class="ri-arrow-right-up-line"></i></span></button>
                 </p>
-                <div class="search">
-                <input type="text" placeholder="Search...">
-                <button class="search-button">Search</button>
+                <div id="searchBox">
+                    <input type="text" id="input" placeholder="Search our blog...">
+                    <button id="searchBtn">Search</button>
                 </div>
             </div>
         </div>
     </div>
     </section>
-    <section>
+    <h2 class="testi-head">What Our Visitors Say?</h2>
+    <section class="testimonials">
         
-    </section>
-    <section class="testimonials-section">
-        <h2>What Our Visitors Say</h2>
-        <div class="testimonial">
-            <p class="quote">"This blog has helped my child immensely with on his computer projects. Their approach is engaging and effective!"</p>
-            <p class="author">– Sarah L., Parent</p>
+        <div class="testimonial-card">
```

Comment:
Very good on applying the changes for this. However, add enough padding for top and bottom for the cards. Overall they look very good.